README.md

```diff
@@ -11,6 +11,26 @@ Data is being uploaded to `s3://autofocus`. Providing convenient ways to downloa
 
 ## Steps for Training a Camera Traps Model
 
+### 0. download_images.py
+
+#### Example call
+```bash
+python autofocus/download_images.py --local_folder local_dir --bucket autofocus
```

Comment:
Let's use `data` as the `--local_folder` argument, and change the subsequent example calls to match. At some point I'd like to have a quickstart version that builds a model on a small subset of the data.